ffs/integrationtest/integration_test.go

```diff
@@ -420,7 +420,7 @@ func TestDurationConfig(t *testing.T) {
 	cinfo, err := fapi.Show(cid)
 	require.Nil(t, err)
 	p := cinfo.Cold.Filecoin.Proposals[0]
-	require.Equal(t, duration, p.Duration)
+	require.Greater(t, p.Duration, duration)
```

Comment:
So, this is considering all the discussion today with the _duration_ in Lotus.
The real/confirmed Deal duration isn't what we proposed in the deal (since, its minDuration). So the confirmed duration of the deal should be greater than the proposed one (not equal).